css-responsive/index.html

```diff
@@ -0,0 +1,441 @@
+          <div>
+            <p class="font-normal tracking-tight leading-tight">Florencia López</p>
+            <p class="font-light tracking-tight leading-snug">Human Resources</p>
+          </div>
+        </div>
+
+      </div>
+      <div class="hidden md-flex flex-col gap-2 w-50 mt-3">
+        <div>
+          <img src="../assets/svg/vector_quote.svg"></img>
+        </div>
+        <hr class="border-t-line">
+        <h6 class="mt-1 font-normal text-md tracking-ticht leading-tight">Muchísimas opciones concentradas en un solo
+          lugar. Poder comparar, elegir y gestionar todo desde la web es increíble.
+        </h6>
+        <div class="mt-3 flex items-center gap-2">
+          <div class=" flex items-center rounded-xl shadow-xl bg-light-gray w-6 h-4">
+            <img src="../assets/images/logo_rootstrap.jpeg" class="w-70 h-70 m-inline"></img>
+          </div>
+          <div>
+            <p class="text-md font-normal tracking-tight leading-tight">Sandra De La Roche</p>
+            <p class="text-md font-light tracking-tight leading-snug">Administration Specialist</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="flex flex-col items-center gap-4 mt-4 md-mt-5 w-90 md-w-80 m-inline">
+    <h2
+      class="text-blue md-text-black text-center font-medium md-font-semibold text-lg md-text-3xl leading-snug md-leading-tight md-tracking-tight w-80 text-center">
+      ¿Tenés
+      un servicio o
+      producto para ofrecer?
+    </h2>
+    <h6
+      class="text-base md-text-md md-tracking-ticht md-leading-tight font-medium md-font-normal w-80 md-w-55 text-center">
+      Si
+      ofreces servicios o productos para
+      empresas, PlanIT es el lugar para vos. Escribí tus datos y nos contactaremos contigo.</h6>
+    <div class="mt-1-5 md-mt-3 flex items-center">
+      <button type="button"
+        class="mt-1 py-0-9 px-4 rounded-lg border-none bg-primary-gradient text-white text-sm md-text-base md-tracking-tight md-leading-tight font-semibold md-font-medium shadow-md cursor-pointer text-uppercase">Quiero
+        ser parte
+      </button>
+    </div>
+  </section>
+
+  <section class="flex bg-blue mt-5 rounded-t text-white ">
+    <div class="w-90 md-w-80 m-inline mt-2 md-mt-4">
+      <div class="flex justify-between">
+        <div class="flex flex-col w-100 md-w-50">
+          <h2 class="font-semibold text-xl md-text-3xl md-tracking-tight md-leading-tight leading-snug">¿Listo para
+            organizarte <span class="text-light-blue"> 10 veces </span> más rápido?
+          </h2>
+          <form class="flex mt-2 md-hidden flex-col w-100">
+            <input type="text" placeholder="Empresa" class="bg-transparent text-white border-bottom"><br><br>
+            <input type="text" placeholder="Email" class="bg-transparent text-white border-bottom"><br><br>
+            <input type="text" placeholder="Mensaje" class="bg-transparent text-white border-bottom"><br><br>
```

Comment:
the border-bottom name is not an indicative of what styles we are applying to the inputs, by the name we can only know it's a border bottom but not how it looks.